eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_worker.rb

```diff
@@ -68,30 +68,31 @@ def start_process(options, queue, block)
           options[:durable] = true
         end
 
-        client = options[:client]
         manager = EventQ::RabbitMq::QueueManager.new
         manager.durable = options[:durable]
-        @connection = client.get_connection
-
         @threads = []
 
         #loop through each thread count
         @thread_count.times do
           thr = Thread.new do
 
+            client = options[:client] || new_client_instance(options) # singleton or non-singleton
```

Comment:
Do we need to be backwards compatible?  I think the risk outweighs the benefit of this option.